### ISSUE TITLE
Downloading and installing screen readers with a script

### ISSUE THREAD

**Issue by zcorpan** [w3c/at-driver#14]
From the ARIA-AT automation meeting on March 28, 2022: w3c/aria-at-automation#17 ([minutes](https://www.w3.org/2022/03/28-aria-at-minutes.html))

There is a need for downloading and installing screen readers with a script (e.g. in a CI environment) (for screen readers that aren't already available on the OS).

- Desire to test different versions of a screen reader in parallel (or sequentially)
- Are there portable executables for each screen reader?
- Can you install simultaneously multiple versions of the same reader?

cc @mcking65 @s3ththompson

**Comment by lolaodelola** [comment]
@jugglinmike My understanding is that this is solved with the new automation tool? Can this issue be closed?

**Comment by jugglinmike** [comment]
@lolaodelola We have not solved this problem[^1], but I'm comfortable closing the issue anyway because it now seems out-of-scope for this repository (this repository is currently focused on defining a document which has no need for such a script).

[^1]: For context: as of January 2024, the ARIA-AT project allows Test Admins to initiate fully-automated AT response collection jobs. It does this by triggering work as defined by [this GitHub repository](https://github.com/bocoup/aria-at-gh-actions-helper) (which includes a complete copy of NVDA that has been manually packaged with [the "AT Driver" add-on for NVDA](https://github.com/Prime-Access-Consulting/nvda-at-automation)).
